batch-inference/README.md

```diff
@@ -0,0 +1,52 @@
+
+
+### About
+
+Example shows mechanics of Batch inference
+
+Instead of using some ML Framework it focuses on ODAHU functionality and replaces
+"real" model by a simple `model` that contains multiplier.txt that defines a `multiplier`.
+Every number in input tensor will be multiplied by `multiplier` and returned as output tensor.
+
+### Example structure:
+
+- `input` defines inference request in [kubeflow format](https://github.com/kubeflow/kfserving/blob/master/docs/predict-api/v2/required_api.md#inference-request-json-object)
```

Comment:
I'd suggest to refer to a fixed tag/branch in `kfserving` repo. They can move/rename this page in `master` and our users will see 404 Not Found.